graylog2-server/src/main/java/org/graylog2/streams/OutputAVImpl.java

```diff
@@ -32,44 +32,52 @@
 @JsonAutoDetect
 @CollectionName("outputs")
 public abstract class OutputAVImpl implements Output {
```

Comment:
I think `Output` exists because it's in the plugin interfaces?
Personally I find questionable whether that module makes so much sense, but that was at least the reason for the interface.